deployment/deployer.yaml

```diff
@@ -67,6 +67,16 @@ items:
       - watch
       - delete
       - update
+  - apiVersion: v1
```

Comment:
We need to be able to label nodes so the fluentd daemonset (un)schedules pods.  Unless there is a verb for just labelling that I'm not aware of?

the deployer is trying to use `oc label nodes --all ${nodeSelectorLabel}-`